src/legacy/ui/public/index_patterns/_field_list.ts

```diff
@@ -18,16 +18,17 @@
  */
 
 import { IndexedArray } from 'ui/indexed_array';
-import { Field } from './_field';
+import { IndexPattern } from 'ui/index_patterns/_index_pattern';
+import { Field, FieldSpec } from './_field';
 
-export class FieldList extends IndexedArray {
-  constructor(indexPattern, specs) {
+export class FieldList extends IndexedArray<Field> {
+  constructor(indexPattern: IndexPattern, specs: FieldSpec[], shortDotsEnable: boolean = false) {
```

Comment:
okay, removed. 👍 

it doesn't make too much of a difference to TS because it is inferred as boolean regardless, but this removes some redundant code.